modules/nbde-rekeying-tang-servers.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// security/nbde-implementation-guide.adoc
+
+[id="nbde-rekeying-tang-servers_{context}"]
+= Rekeying Tang servers
+
+This procedure uses a set of three Tang servers, each with unique keys, as an example.
+
+Using redundant Tang servers reduces the chances of nodes failing to boot automatically.
+
+.Prerequisites
+
+* A working Network-Bound Disk Encryption (NBDE) encryption on the Tang server.
```

Comment:
```suggestion
* A working Network-Bound Disk Encryption (NBDE) on the Tang server.
```

It said encryption encryption - didn't know if that was by mistake. Ignore if it was previously correct :-)

Reply:
Yes, it does seem awkward, but it's first occurrence of NBDE, so...

Reply:
Changed as per Jim Ramsey to " A working Network-Bound Disk Encryption (NBDE) installation on one or more nodes."